pkgs/tools/package-management/createrepo_c/default.nix

```
@@ -1,18 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, bzip2, expat, glib, curl, libxml2, python3, rpm, openssl, sqlite, file, xz, pcre, bash-completion }:
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, bzip2, expat, glib, curl, libxml2, python3, rpm, openssl, sqlite, file, xz, pcre, bash-completion, zstd, zchunk, libmodulemd }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, bzip2, expat, glib, curl, libxml2, python3, rpm
, openssl, sqlite, file, xz, pcre, bash-completion, zstd, zchunk, libmodulemd
}:
```

that line got a bit long.